firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -69,33 +69,11 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
                     TabContentBlocker.prefsChanged()
                     self.tableView.reloadData()
 
-                    let extras = [
-                        TelemetryWrapper.EventExtraKey.preference.rawValue: "ETP-strength",
-                        TelemetryWrapper.EventExtraKey.preferenceChanged.rawValue: option.rawValue
-                    ]
-                    TelemetryWrapper.recordEvent(
-                        category: .action,
-                        method: .change,
-                        object: .setting,
-                        extras: extras
-                    )
-
                     if option == .strict {
                         self.linkButton.isHidden = true
```

Comment:
Question for experts: Does this work as intended? I tried running the app and expected the link button to be hidden, but it seems it's not working. Is this a bug, or am I missing something?